### PR TITLE
updata the project section and teammembers completely.

### PR DESCRIPTION
**Pull request by cainiaogoroad**
1.add the imformation of mayinzi.
2. fix the error of m-LoRA.md can't display image.
3.Icons were added to the corresponding items, except for mlora, which was not added as there was no icon

### CODE REVIEW

File: _projects/m-LoRA.md
Comment (youxingling) [inline _projects/m-LoRA.md:1]: Please update icon for m-LoRA, including modify `m-LoRA.md` and upload a icon. 
The same goes for couler.

Comment (cainiaogoroad) [reply _projects/m-LoRA.md:1]: m-LoRA have not icon,what shuold i do

Comment (cainiaogoroad) [reply _projects/m-LoRA.md:1]: m-LoRA don't have icon,i can't update icon for it.

Comment (youxingling) [reply _projects/m-LoRA.md:1]: Sorry, I've seen your comment. Please modify `couler.md` to update its icon.

**Pull request closed without merging**